packages/twenty-front/src/modules/object-record/cache/utils/getRecordNodeFromRecord.ts

```diff
@@ -65,20 +65,23 @@ export const getRecordNodeFromRecord = <T extends ObjectRecord>({
           return undefined;
         }
 
-        if (Array.isArray(value)) {
-          const objectMetadataItem = objectMetadataItems.find(
-            (objectMetadataItem) => objectMetadataItem.namePlural === fieldName,
+        if (
+          Array.isArray(value) &&
```

Comment:
let's re-write this. The conversion from record to recordNode should be completely piloted by Metadata. the Array case is actually the case of fieldType = RELATION (oneToMany)

Reply:
ok, wasn't sure so that is nice to use RELATION I agree